analyzer/src/main/kotlin/managers/Cargo.kt

```diff
@@ -266,12 +266,14 @@ class Cargo(
                 }
                 .toSortedSet()
 
-            return Scope(name = scope, dependencies = transitiveDependencies)
+            return if (transitiveDependencies.isEmpty()) null else Scope(scope, transitiveDependencies)
```

Comment:
IIRC we don't skip empty scopes for other package managers because it is still useful information that a scope exists but does not contain any dependencies, or do we? If we change it it should be consistent across package managers, but I would prefer to keep it as is.

Reply:
I'll edit the commit to make more clear what this is about: It's actually not about *empty* scopes, but about *non-existing* scopes. As you can see e.g. in `analyzer/src/funTest/assets/projects/synthetic/cargo-subcrate/client/Cargo.toml` only the `dependencies` scope is declared, but still all scopes searched by ORT are listed in the result, which is wrong, IMO.